### PR TITLE
fixing the search for addresses with no transactions

### PR DESCRIPTION
**Pull request by McEdward**
This should fix the new issues with #255

### CODE REVIEW

File: db/dcrsqlite/apisource.go [hunk 1]
```diff
@@ -1090,7 +1090,8 @@ func (db *wiredDB) GetExplorerAddress(address string, count, offset int64) *expl
 			return nil
 		}
 		return &explorer.AddressInfo{
-			Address: address,
+			Address:  address,
+			MaxLimit: maxcount,
```
Comment (chappjc) [inline db/dcrsqlite/apisource.go]: I'd like for us to start using more descriptive variable names. E.g. what is this a limit on? Also, how are Max and limit not redundant?  I'm guilty of this too btw

File: explorer/explorerroutes.go [hunk 2]
```diff
@@ -217,15 +217,19 @@ func (exp *explorerUI) AddressPage(w http.ResponseWriter, r *http.Request) {
...
 			pageData := struct {
 				Data          *AddressInfo
 				ConfirmHeight []int64
+				Version       string
 			}{
 				addrData,
-				nil,
+				confirmHeights,
+				exp.Version,
 			}
```
Comment (chappjc) [inline explorer/explorerroutes.go]: Good catch

File: db/dcrsqlite/apisource.go (hunk 1 above)
Comment (McEdward) [reply db/dcrsqlite/apisource.go]: Yes, limit and maxlimit look redundant but limits value changes from the template not Maxlimit which is always the upper limit of transactions that's can be possible in litemode

Comment (chappjc) [reply db/dcrsqlite/apisource.go]: The key word there is "transactions".  What is it a limit on?

File: explorer/explorerroutes.go [hunk 3]
```diff
@@ -271,6 +275,8 @@ func (exp *explorerUI) AddressPage(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	log.Info(addrData.KnownFundingTxns, addrData.MaxLimit)
```
Comment (chappjc) [inline explorer/explorerroutes.go]: Please remove

**Comment by chappjc** [comment]
good for merge if anyone gives a tACK

**Comment by RogueElement** [comment]
full mode:
screen shot 2018-02-09 at 19 31 31\
lite mode:
screen shot 2018-02-09 at 19 32 39

**Pull request merged**